utils/log4cplus/log_utils.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2011-2020 Intel Corporation. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *   * Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in
+ *     the documentation and/or other materials provided with the
+ *     distribution.
+ *   * Neither the name of Intel Corporation nor the names of its
+ *     contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+// #ifndef _LOG_UTILS_H
+// #define _LOG_UTILS_H
+
+#include <log4cplus/logger.h>
+#include <log4cplus/loggingmacros.h>
+#define IS_DEBUG true
+
+using namespace log4cplus;
+using namespace log4cplus::helpers;
+
+#if defined(__linux__)
+    #define VSPRINTF vsnprintf
+#elif defined(_WIN32)
+    #define VSPRINTF _vsnprintf
+#endif
+
+// global object
+extern Logger logger;
+
+// define some macros for simplicity
+void log_d(const char* pFormat,  ...);
+void log_i(const char* pFormat,  ...);
+void log_e(const char* pFormat,  ...);
+void log_w(const char* pFormat,  ...);
+
+void initLogger(bool isDebug);
+ 
+void shutDown();
+ 
+void close();
```

Comment:
anywhere close() and shutDown() are called?
And I didn't find the definition of close().

Reply:
done.